docs/basics/errors.md

```diff
@@ -55,150 +62,180 @@ bad meta data in file
 
 :   The compressed file contains corrupt meta data. This can happen if the file was incomplete at the time of reading.
 
+[](){#badtail}
 badtail
 
 :   Incomplete transaction at end of file, get good (count;length) with ``-11!(-2;`:file)``
 
+[](){#binary-mismatch}
 binary mismatch
 
 :   Wrong process for [code profiler](../kb/profiler.md)
 
+[](){#can't}
 can't
 
 :   Only commercially licensed kdb+ instances can encrypt code in a script
 
+[](){#cast}
 cast
 
 :   E.g. ``s:`a`b; c:`s$`a`e``
 
     Value not in enumeration
 
+[](){#close}
 close
 
 :   (1) content-length header missing from HTTP response
 
     (2) handle: n – handle was closed by the remote while a msg was expected
 
+[](){#con}
 con
 
 :   qcon client is not supported when kdb+ is in [multithreaded input mode](../kb/multithreaded-input.md)
 
+[](){#cond}
 cond
 
 :   Even number of arguments to `$` (until V3.6 2018.12.06)
 
+[](){#conn}
 conn
 
 :   Too many connections. Max connections was 1022 prior to 4.1t 2023.09.15, otherwise the limit imposed by the operating system (operating system configurable for system/protocol).
 
+[](){#Could-not-initialize-ssl}
```

Comment:
Is this one really a capital letter when the rest are not?

Reply:
changed now to lowercase